### PR TITLE
Add and display tasks

### PR DESCRIPTION
**Pull request by hoangtv2605**

### CODE REVIEW

**Comment by hoangtv2605** [comment]
ready

File: quickstart/resources/views/common/errors.blade.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@if (count($errors) > 0)
+<div class="alert alert-danger">
+    <strong>{{ trans('error.message') }}</strong>
+
+    <br><br>
+
+    <ul>
+        @foreach ($errors->all() as $error)
+        <li>{{ $error }}</li>
```
Comment (huyenttm-0901) [inline quickstart/resources/views/common/errors.blade.php]: indent 4 spaces

**Comment by hoangtv2605** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
merged

**Pull request merged**